app/models/user.rb

```diff
@@ -47,4 +47,20 @@ class User < ApplicationRecord
   after_update if: [:verified_previously_changed?, :verified?] do
     events.create! action: "email_verified"
   end
+
+  def self.from_omniauth(access_token)
```

Comment:
```suggestion
  def self.from_omniauth(auth)
```

Feels a little like _`_token`_ isn't quite right here.